### ISSUE TITLE
Older version Netlify links broken

### ISSUE THREAD

**Issue by thiagodebastos** [reactjs/reactjs.org#4306]
I have been unable to locate documentation for React ^16.8. After further investigation I noticed that every documentation link I attempted to access, led to a Netlify error. For example https://reactjs.org/version/16.8:

```
Not Found - Request ID: 01FV995X72RZM92K5WZM6J14AV
```

![image](https://user-images.githubusercontent.com/6151341/152727646-e3d69e1b-d056-4f8c-852f-a4824d1fb7cc.png)

**Comment by gaearon** [comment]
Yeah, we need to republish them somewhere. Initially we probably thought these deployments would last forever, but they didn't. :)

**Comment by ashutosh887** [comment]
Should I work on this Issue @gaearon

**Comment by thiagodebastos** [comment]
> Yeah, we need to republish them somewhere. Initially we probably thought these deployments would last forever, but they didn't. :)

Thanks for the context :)
